pkgs/by-name/fy/fyne/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+
+  dbus,
+  libGL,
+  libX11,
+  libXcursor,
+  libXinerama,
+  libXi,
+  libXrandr,
+  libXxf86vm,
+  pkg-config,
+}:
+
+let
+  version = "2.4.4";
+in buildGoModule {
+  pname = "fyne";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "fyne-io";
+    repo = "fyne";
+    rev = "v${version}";
+    hash = "sha256-6gDHURcf4dVmaiXleeRKogAtCAgcHU437TjEHhfGdj8=";
+  };
+
+  vendorHash = "sha256-0P2YqXq+c20F2hVkRcvwV5PpqUoZ3PnRIvTHt/qAMPA=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libGL
+    libX11
+    libXcursor
+    libXinerama
+    libXi
+    libXrandr
+    libXxf86vm
+  ];
+
+  checkInputs = [
+    dbus
+  ];
+
+  doCheck = true;
+
+  preCheck = ''
+    OLD_DIR=$PWD
+    OLD_GOPATH=$GOPATH
+    export HOME=$(mktemp -d)
+    export GOPATH=$(mktemp -d)
+    cd $HOME
```

Comment:
not sure this is right

Reply:
It's unfortunately necessary for tests to pass.